lms/templates/discussion/_thread_list_template.html

```diff
@@ -40,7 +41,8 @@
                         <option value="flagged">${_("Flagged")}</option>
                     %endif
                 </select>
-            </label>${"<% if (isCohorted && isPrivilegedUser) { %>"}<label class="forum-nav-filter-cohort">
+            ## safe-lint: disable=python-parse-error,python-wrap-html
```

Comment:
Reviewers: These were the only two false positives which resulted in a parse error that I disabled.